masterlist.yaml

```diff
@@ -7084,12 +7099,43 @@ plugins:
       # version: 1.58
       - crc: 0xD7423A4E
         util: 'SSEEdit v4.0.0'
+  - name: 'imp_helm_Immersive_Armors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
+    after:
+      - 'Hothtrooper44_ArmorCompilation.esp'
+    clean:
+      # version: 1
+      - crc: 0x5C4A5C28
+        util: 'SSEEdit v4.0.2'
   - name: 'imp_helm_IDPM.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
         name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
     after:
       - 'imp_helm_USSEP.esp'
+      - 'Improved Dragon Priest Masks.esp'
```

Comment:
Neither rule is required as `imp_helm_legend.esp` and `Improved Dragon Priest Masks.esp` are masters of `imp_helm_IDPM.esp`.

![image](https://user-images.githubusercontent.com/1944639/57930917-177a1f00-78af-11e9-86de-6d445959f829.png)